src/ui/common/src/stories/Button.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
```

Comment:
what's the relationship between these component with those in `primitives`? Which ones should I use?

Reply:
First off, this component is just here to demonstrate how StorybookJS works. 
The button component here comes from the default installation of StorybookJS.

So, we shouldn't use these over the components that are defined in primitives. Once I get some stories working for our own components we can come back and delete this component along with the other default stories and components that Storybook gives us.